src/core/Expander/Expander/Expander.md

```diff
@@ -80,64 +243,235 @@ import {
+        organisations that use the service.
+        <ul>
+          <li>
+            The messages and attachments shown in the application are
+            transmitted in encrypted form between the mobile device
+            and the server.
+          </li>
+          <li>
+            The app requires permissions to use your mobile device’s
+            photographs and other files, so it can process attachments
+            included in messages.
+          </li>
+          <li>
+            The user is individually identified using strong
+            identification. This means that you are the only person
+            who can access your data.
+          </li>
+        </ul>
+      </Paragraph>
+      <Paragraph>
+        If you break your mobile device or lose it, you can sign out
+        of the Suomi.fi application either in the Suomi.fi web service
+        or by using another phone.
+      </Paragraph>
+    </ExpanderContent>
   </Expander>
   <Expander>
-    <ExpanderTitleButton>Test expander 3</ExpanderTitleButton>
-    <ExpanderContent>Test expander content 3</ExpanderContent>
+    <ExpanderTitleButton>
+      Errors in the Suomi.fi mobile application
+    </ExpanderTitleButton>
+    <ExpanderContent>
+      <Paragraph marginBottomSpacing="l">
+        If the application does not work on your mobile device for
+        some reason, you can also read your messages via a browser
+      </Paragraph>
+      <Paragraph>
+        Errors occurring in the Suomi.fi application generally do not
+        affect the preservation of Suomi.fi Messages.
+      </Paragraph>
+    </ExpanderContent>
   </Expander>
 </ExpanderGroup>;
 ```
 
-## Controlled
+### Controlled state
 
-- State for the individual Expanders are stored outside of the component and user has full control.
-- Therefore when clicking the individual Expander they are not opened by default, user have to give the logic to change it.
-- It's user's responsibility to keep the state stored outside to be updated as Open/Close All is used.
-- `defaultOpen` prop will not work when Expander is in controlled state == `open` prop is given.
+You can control the state of an individual Expander programmatically with the `open` prop.
 
 ```jsx
 import {
   Expander,
   ExpanderGroup,
   ExpanderTitleButton,
-  ExpanderContent
+  ExpanderContent,
+  Paragraph,
+  Text
 } from 'suomifi-ui-components';
+import { useState } from 'react';
 
-const [expanderThreeOpen, setExpanderThreeOpen] =
-  React.useState(false);
+const [expanderOneOpen, setExpanderOneOpen] = useState(false);
+const [expanderTwoOpen, setExpanderTwoOpen] = useState(false);
+const [expanderThreeOpen, setExpanderThreeOpen] = useState(true);
 
 <>
+  <button onClick={() => setExpanderTwoOpen(!expanderTwoOpen)}>
+    Toggle middle expader
+  </button>
   <ExpanderGroup
     openAllText="Open all"
     ariaOpenAllText="Open all expanders"
     closeAllText="Close all"
     ariaCloseAllText="Close all expanders"
```

Comment:
Because I doubt "expanders" is ever a word any user would understand, I suggest using "sections" as a default suggestion.
```suggestion
    ariaOpenAllText="Open all sections"
    closeAllText="Close all"
    ariaCloseAllText="Close all sections"
```

Reply:
An excellent point 👍